Headers/DebugServer2/GDBRemote/DummySessionDelegateImpl.h

```diff
@@ -252,8 +252,8 @@ class DummySessionDelegateImpl : public SessionDelegate {
 
   ErrorCode onQueryProcessList(Session &session, ProcessInfoMatch const &match,
                                bool first, ProcessInfo &info) const override;
-  ErrorCode onQueryProcessInfo(Session &session, ProcessId pid,
-                               ProcessInfo &info) const override;
+  ErrorCode onQueryProcessInfoPID(Session &session, ProcessId pid,
```

Comment:
Why not use an overloaded name?

Reply:
That's what it was originally, once I started using the new `onQueryProcessInfo` function in the session implementation, clang gave me `-Woverloaded-virtual` warnings (on macos only for some reason). Renaming was the straight-forward fix. And, honestly, I think this naming is better because it is consistent with the packet naming, `qProcessInfo` and `qProcessInfoPID`.